sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -48,6 +48,7 @@ enum class architecture : uint64_t {
   //       - DD is 2-digit number reserved for future unexpected modifications
   //         to keep uniqueness. It should be always 00 for now
   //
+  unknown = 0x9800000000000000,
   x86_64 = 0x9900000000000000,
```

Comment:
@gmlueck I also found one discrepancy between spec and impl: spec says that `x86_64` is a part of `Intel CPU family`. Should we put `x86_64` to `Intel CPU family` if it is also applicable for e.g., AMD CPUs?

Reply:
Yes.  We will probably make some more changes to the CPU architecture enums in the future, but I think this makes sense for now.

Reply:
Ok, I will update the impl then and move `x86_64` under the Intel CPU family

Reply:
Applied in https://github.com/intel/llvm/pull/14077/commits/e4ce1080abce5fdbb8de271755b36d48f9f1f718